test/common/grpc/google_grpc_creds_test.cc

```diff
@@ -0,0 +1,48 @@
+#include "common/grpc/google_grpc_creds_impl.h"
+
+#include "test/common/grpc/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Grpc {
+namespace {
+
+// Force execution of all paths in Grpc::defaultChannelCredentials(). Since the underlying
+// grpc::{CallCredentials,ChannelCredentials} don't have any real way of getting at the underlying
+// state, we can at best just make sure we don't crash.
+TEST(DefaultChannelCredentials, All) {
+  { EXPECT_NE(nullptr, defaultChannelCredentials({})); }
```

Comment:
since default is `true` (before my comment above) these 2 lines are testing same.